src/cli/providers/messenger/whitelisted-domains.js

```diff
@@ -34,13 +34,13 @@ const help = () => {
 };
 
 export async function getWhitelistedDomains(ctx) {
-  const { t, token: _token } = ctx.argv;
+  const { t, token } = ctx.argv;
```

Comment:
How about 

```js
const token = ctx.argv.token || ctx.argv.t;
```

Reply:
ping @kpman

Reply:
Inspired by this idea, I left some comments here https://github.com/Yoctol/bottender/pull/349#discussion_r228774753